plugins/notifications/config.d.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2020 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export interface Config {
+  /** @visibility frontend */
+  notifications?: {
+    /**
+     * Enable or disable the Web Notification API for notifications, defaults to false
+     * @visibility frontend
+     */
+    enableWebNotifications?: boolean;
+    /**
+     * Enable or disable the title override to show notification count, defaults to true
+     * @visibility frontend
+     */
+    enableTitleCounter?: boolean;
```

Comment:
With the new frontend system in mind we'll most likely move away from this kind of global config. It'd be possible to just leave this as props for the sidebar item for now right?